docs/posters-and-talks.rst

```diff
@@ -3,17 +3,47 @@ Posters and Talks
 
 This page includes various talks and posters that the Dockstore team has given.
 
+2020
+----
+
+BOSC 2020
+^^^^^^^^^
+Cabansay L, Duncan A and Yuen D. Dockstore fundamentals: Introduction to docker and descriptors for reproducible analysis [version 1; not peer reviewed]. F1000Research 2020, 9(ISCB Comm J):986 (slides) (https://doi.org/10.7490/f1000research.1118245.1)
+
+- :download:`Workshop <https://bcc2020.sched.com/event/c46B/dockstore-fundamentals-introduction-to-docker-and-descriptors-for-reproducible-analysis>`
+- :download:`Youtube <https://www.youtube.com/watch?v=shMr_Bd01Ko>`
+
+Reproducible Analysis in the Cloud with Dockstore and Terra
+
+- `Workshop <https://bcc2020.sched.com/event/c4pR/reproducible-analysis-in-the-cloud-with-dockstore-and-terra>`_
 
 
 2019
 ----
 
+Youtube
+^^^^^^^
+
+Dockstore: An Introduction
+
+- `Youtube <https://www.youtube.com/watch?v=1JWOVGzzgMc>`_
+
+CCRC 2019
+^^^^^^^^^
+
+Yuen, Denis, Baker, Jared, Duncan, Andrew, & Brazas, Michelle. (2019, November). Using Clouds for Big Cancer Data Analysis. Zenodo. http://doi.org/10.5281/zenodo.4527274
```

Comment:
Should it be `Yuen Denis, Baker Jared,...` instead of also having commas after the last names? Otherwise they all blend together.

Reply:
Hah, think I misunderstood the zenodo author entry form